modules/persistent-storage-flexvolume-installing.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-flexvolume.adoc
+
+[id="flexvolume-installing_{context}"]
+
+= Installing FlexVolume drivers
+
+FlexVolume drivers that are used to extend {product-title} are executed only on the node. To implement FlexVolumes, a list of operations to call and the installation path are all that is required.
+
+.Prerequisites
+
+* FlexVolume drivers must implement these operations:
+
+`init`::
+Initializes the driver. It is called during initialization of all nodes.
+
+** Arguments: none
+** Executed on: node
+** Expected output: default JSON
+
+`mount`::
+Mounts a volume to directory. This can include anything that is necessary
+to mount the volume, including attaching the volume to the node, finding the device, and then mounting the device.
+
+** Arguments: `<mount-dir>` `<json>`
+** Executed on: node
+** Expected output: default JSON
+
+`unmount`::
+Unmounts a volume from a directory. This can include anything that is necessary to clean up the volume after unmounting, such as detaching the volume
+from the node.
+
+** Arguments: `<mount-dir>`
+** Executed on: node
+** Expected output: default JSON
+
+`mountdevice`::
+Mounts a volume's device to a directory. `<device-name>` is name of the device as returned by the previous `waitforattach` operation.
+
+** Arguments: `<mount-dir>` `<device-name>` `<json>`
+** Executed on: node
+** Expected output: default JSON
+
+`unmountdevice`::
+Unmounts a volume's device from a directory.
+
+** Arguments: `<mount-dir>`
+** Executed on: node
+** Expected output: default JSON
+
+* All other operations should return JSON with `{"status": "Not supported"}` and exit code `1`.
+
+.Procedure
+
+To install the FlexVolume driver:
+
+. Ensure that the executable file exists on all masters and nodes in the cluster.
+
+. Place the executable file at the volume plug-in path:
+*_/etc/kubernetes/kubelet-plugins/volume/exec/<vendor>~<driver>/<driver>_*.
```

Comment:
@gnufied  I believe these install paths were updated since this: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-storage/flexvolume.md#prerequisites
Can you confirm that this install path and the one below [52] are correct?